resources/leiningen/new/xiana/test/app_name_fixture.clj

```diff
@@ -23,24 +23,16 @@
     (jdbc/execute! (dissoc db-config :dbname) [init-sql])
     (assoc config :framework.db.storage/postgresql db-config)))
 
-(defn migrate!
-  [config]
-  (let [db (:framework.db.storage/postgresql config)
-        mig-config (assoc (:framework.db.storage/migration config) :db db)]
-    (migratus/migrate mig-config))
-  config)
-
```

Comment:
Just Frankie doesn't use migratus right now, but this should be here at default

Reply:
https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/test/app_name_fixture.clj#L27-L32